conf/application.conf

```
@@ -129,7 +129,7 @@ microservice {
 
         file-upload {
             host = localhost
-            port = 9203
+            port = 8899
```

This is not the right port.
```suggestion
            port = 8898
```